include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -333,4 +333,11 @@
 #    define _ONEDPL_STD_RANGES_ALGO_CPP_FUN 0
 #endif
 
+#if _ONEDPL_STD_RANGES_ALGO_CPP_FUN
+#   define _ONEDPL_ALGO(std_algo_name) \
+[](auto&&... __args) { return std_algo_name(std::forward<decltype(__args)>(__args)...); }
```

Comment:
Will not it be better to move that utility in a separate header, e.g. `utils.h`? 
A "config" usually contains well some settings, e.g. to enable workarounds, rather than workarounds themselves. Also, having a macro (which are generelly avoided) looks unnecessary to resolve that particular issue.

Reply:
To tell the truth, practically, many macro wrappers were accumulated in `onedpl_config.h`. F.e. a lot of _ONEDPL_PRAGMA_SIMD wrappers.
`utils.h`, traditionally, contains C/C++ functons/utilities.

Reply:
I  implied creation of a function instead of a macro in `utils.h`, but perhaps it is a matter of taste, so I do not insist.

Reply:
No, it is not "it is a matter of taste"...
Of course I prefer function too... But there is no way to write a function here.

Reply:
But it is possible to write a function in `utils.h`, if I understood you correctly.

Reply:
I think I agree that this is more of a "utility" than a "configuration".  I understand the motivation to group preprocessor macros together, and utility functions together, but the concept seems more important to me.  
I think I would be in favor of keeping `_ONEDPL_STD_RANGES_ALGO_CPP_FUN` definition here, and moving the rest to "utils.h", but I also wont insist.

Reply:
I don't mind

Reply:
After taking a closer look, and I cannot come up with a solution without macros. Given that, I agree keeping it in `onedpl_config.h`.

Reply:
I'm OK with whatever you decide.